### ISSUE TITLE
Disabled buttons have low contrast for text and borders

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#6221]
.NET Core Version: 
- .NET Core SDK: 7.0.0-alpha.1.21567.1

Have you experienced this same bug with .NET Framework?: 
- Yes

**Repro steps:**
1. Create a form with a button
2. Make button disabled
3. Run the application

**Notes:**
The issue is reproduced for high contrast mode.

**Actual behavior:**
Flat button text:
![image](https://user-images.githubusercontent.com/23376742/143261747-1a07d49a-d62c-4d98-b0d2-e9705c07fb8b.png)

Popup button text:
![image](https://user-images.githubusercontent.com/23376742/143262223-6fe99c91-70eb-446c-a3ad-d42b01b63a7b.png)

Popup button border:
![image](https://user-images.githubusercontent.com/23376742/143262557-6cadcb01-1a57-448c-ad4e-a16ca891fc0f.png)

Standard button text:
![image](https://user-images.githubusercontent.com/23376742/143263180-4c89942d-9379-4ec1-96f0-811dcaaf10ac.png)

Standard button border:
![image](https://user-images.githubusercontent.com/23376742/143262844-3881b2c8-d924-459f-8ed6-091e6e32d46c.png)

System button text:
![image](https://user-images.githubusercontent.com/23376742/143263358-48512cf7-8587-4893-aa46-23b0c96e1a3a.png)

System button border:
![image](https://user-images.githubusercontent.com/23376742/143263749-e8e7571b-17d6-45c4-82a3-dadeb6b0b50a.png)

**Expected behavior:**
Disabled buttons should have correct contrast for text and borders

**Comment by Tanya-Solyanik** [comment]
@SergeySmirnov-Akvelon  - please add the doc link that explains that disabled UI is not required to be contrasting and close this bug

**Comment by vladimir-krestov** [comment]
Disabled controls are not required to be contrast:
- "_User Interface Components that are not available for user interaction (e.g., a disabled control in HTML) are not required to meet contrast requirements in WCAG 2.1._" - from the [doc](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#:~:text=Inactive%20User%20Interface%20Components).
- "_Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement._" - from the [doc](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#:~:text=Text%20or%20images%20of%20text%20that%20are%20part%20of%20an%20inactive%20user%20interface%20component%2C%20that%20are%20pure%20decoration%2C%20that%20are%20not%20visible%20to%20anyone%2C%20or%20that%20are%20part%20of%20a%20picture%20that%20contains%20significant%20other%20visual%20content%2C%20have%20no%20contrast%20requirement.).

So closed this Issue.
